ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.12.yaml

```diff
@@ -21,11 +21,6 @@ operator:
   substitutions:
   - pullspec: quay.io/konveyor/oadp-operator:latest
     with: oadp-operator
-promotion:
-  excluded_images:
-  - oadp-operator
-  name: test-oadp-operator-master
-  namespace: openshift
```

Comment:
Why are we removing this?

I think we should be adding this to all but one config for a branch.

Reply:
I wanted to have all configs equal. What this promotion does?

Reply:
alright

so, should we add promotion to the highest versions? In this case, 4.13 and 4.14
(why not add to each version?)

Reply:
because the built images are identical in each "cluster version". Only one is needed.

Reply:
oh instead of adding to 2 of 3, just add to one then?
If so, will add to 4.14, just because is the highest

Reply:
I believe above is "exclusion" so exclude all but one.

Reply:
ohhhh yeah, I did not saw excluded there

so just to confirm

we want ONE promoted image

so, we add 
```yaml
promotion:
  excluded_images:
```
to ALL configs, except ONE

Going to let 4.14 be the promoted one, just because is the highest version

Do we agree on this?

Reply:
> Going to let 4.14 be the promoted one, just because is the highest version

Yes. Sounds good!

Reply:
Still confused

To work, I had to add this to 4.12 and 4.13 (also adding `test-oadp-operator-master` to excluded)
```yaml
promotion:
  excluded_images:
  - oadp-operator
  - test-oadp-operator-master
  name: test-oadp-operator-master
  namespace: openshift
```
If not: `level=error error="output tag openshift/test-oadp-operator-master:test-oadp-operator-master is promoted from more than one place: openshift/oadp-operator@master [4.12], openshift/oadp-operator@master [4.13]"`

and from this log shouldn't we add what we had only in 4.14?

Reply:
perhaps. I haven't looked in detail the exact way to configure promotion. Perhaps its already smart enough when not included to only promote once from one job automatically?

Reply:
as discussed in slack, I am resolving this one